BlueMapCore/src/main/java/de/bluecolored/bluemap/core/webserver/HttpConnection.java

```diff
@@ -66,6 +72,20 @@ public void run() {
 				HttpRequest request = acceptRequest();
 				HttpResponse response = handler.handle(request);
 				sendResponse(response);
+                                if (verbose) {
+                                    DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
+                                    Date date = new Date();
+                                    Logger.global.logInfo(
+                                            connection.getInetAddress().toString() +
+                                            " [ " +
+                                            dateFormat.format(date) +
+                                            " ] \"" +
+                                            request.getMethod() +
+                                            " " + request.getPath() +
+                                            " " + request.getVersion() +
+                                            "\" " +
+                                            response.getStatusCode().toString());
```

Comment:
I think this should be in a new method `if (verbose) log(request, response)` to not clutter the main connection-loop :)

Reply:
No problem. I'll get this done soon.